src/common/statement.cpp

```diff
@@ -51,6 +51,17 @@ void Statement::ParseQueryTypeString(const std::string& query_string,
   boost::to_upper(query_type_string);
 }
 
+void Statement::ParseCreateTypeString(const std::string& query_string,
+                                     std::string& create_type_string) {
+  std::stringstream stream(query_string);
+  std::string query_type_string;
+  stream >> query_type_string;
+  boost::to_upper(query_type_string);
+  PL_ASSERT(query_type_string == "CREATE");
```

Comment:
Can we remove line 57-59 and put `PL_ASSERT(create_type_string == "CREATE");` after line 62? The line 57-59 look redundant to me.

Reply:
query_type_string should be "CREATE" and create_type_string should be "TABLE" or "INDEX". I can remove the variable query_type_string and read into create_type_string twice. Is that what you mean?

Reply:
I see. This is going to be ok for now. 👍 Thanks for the explanation.